cloud/kubernetes/design/operator_design.md

```diff
@@ -0,0 +1,129 @@
+# YugabyteDB Operator Design
+
+A cluster of YugaByte DB should be created using the `ybclusters.yugabyte.com` custom resource definition. Below is a sample custom resource spec for creating a 3-master, 3 t-servers cluster using the CRD. The sample is followed by an explanation of different configuration options available on the YugaByte DB CRD.
+
+
+```yaml
```

Comment:
Let's also have a super minimal CR example - where rest of values are assumed to defaults

Reply:
Done